db/migrations/0002_alter_skill_name.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 4.0.2 on 2024-10-23 13:59
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('db', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='skill',
+            name='name',
+            field=models.CharField(choices=[('Elf', 'Elf'), ('Dwarf', 'Dwarf'), ('Human', 'Himan'), ('Ork', 'Ork')], max_length=255, unique=True),
```

Comment:
There is a typo in the choices for the 'name' field. 'Himan' should be corrected to 'Human'.